test/ssl_test_ctx.c

```diff
@@ -99,6 +99,7 @@ static const test_enum ssl_test_results[] = {
     {"ServerFail", SSL_TEST_SERVER_FAIL},
     {"ClientFail", SSL_TEST_CLIENT_FAIL},
     {"InternalError", SSL_TEST_INTERNAL_ERROR},
+    {"FirstHandshakeFailed", SSL_TEST_FIRST_HANDSHAKE_FAILED},
```

Comment:
What is this?

Reply:
It allows us to identify if the first handshake failed in a resumption test case. Specifically, in handshake_helper.c: do_handshake(). Looks as though this was missing from a prior commit 590ed3d7.